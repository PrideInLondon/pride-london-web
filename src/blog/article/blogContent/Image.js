import React from 'react'
import PropTypes from 'prop-types'
import { ImageContainer, StyledImage } from './Image.styles'

const Image = ({ file }) => (
  <ImageContainer>
    <StyledImage src={file['en-GB'].url} />
  </ImageContainer>
)

Image.propTypes = {
  file: PropTypes.object.isRequired,
}

export default Image
