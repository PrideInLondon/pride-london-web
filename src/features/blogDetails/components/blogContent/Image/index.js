import React from 'react'
import PropTypes from 'prop-types'
import { ImageContainer, StyledImage } from './styles'

const Image = ({ file }) => (
  <ImageContainer>
    <StyledImage src={file['en-GB'].url} />
  </ImageContainer>
)

Image.propTypes = {
  file: PropTypes.object.isRequired,
}

export default Image
