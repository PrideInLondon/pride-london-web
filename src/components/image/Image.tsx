import React from 'react'
import { ImageProps } from './Image.types'

// eslint-disable-next-line jsx-a11y/alt-text
export const Image: React.FC<ImageProps> = props => <img {...props} />
