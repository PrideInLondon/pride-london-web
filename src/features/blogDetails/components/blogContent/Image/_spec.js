import React from 'react'
import { shallow } from 'enzyme'
import Image from '.'

describe(Image.name, () => {
  const fields = { file: { 'en-GB': { url: 'src' } } }
  it('renders with default values', () => {
    const wrapper = shallow(<Image {...fields} />)
    expect(wrapper).toMatchSnapshot()
  })
})
