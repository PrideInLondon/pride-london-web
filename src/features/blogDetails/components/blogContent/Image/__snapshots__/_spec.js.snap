// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image renders with default values 1`] = `
<styles__ImageContainer>
  <styles__StyledImage
    src="src"
  />
</styles__ImageContainer>
`;
